### PR TITLE
Rake db schema load vs rake db migrate

### PR DESCRIPTION
**Pull request by Ikera**
@basti @JovicaSusa @codalicious @draganf @marijanam Added new post rake db:schema:load vs rake db:migrate.

### CODE REVIEW

File: source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  - Ruby and Rails
+tags:
+  - ruby
+  - rails
+  - rake task
+  - globalize
+published: true
+---
+
+Sooner or later every new ruby developer need to understand  differences between this two common rake task. Basically, simple definition tell us everything we need to know:
```
Comment (codalicious) [inline source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown]: Just a few grammar issues I noticed, it should be: 'every new ruby developer need**s**', 'this two common rake task**s**' and '**a** simple definition tell**s** us'

Comment (JovicaSusa) [inline source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown]: @ikere I think that ruby should be capitalised and rake task written in plural.

File: source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
```
Comment (JovicaSusa) [inline source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown]: @ikera I'm not sure that "Basically, simple definition tell..." sounds right, I think that "Basically, these simple definitions... is more appropriate.

File: source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
++ `rake db:migrate` runs migrations that have not run yet
++ `rake db:schema:load` loads the schema.db file into database.
+
+but the right question is when to use one or the other.
+
+**Advice:** <a id="advice"></a> when you adding new migration to a existing app then you need to run `rake db:migrate`, but when you join to existing application (_especially some old application_), or when you drop your applications database and you need to create it again, always run `rake db:schema:load` to load schema.
```
Comment (JovicaSusa) [inline source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown]: @ikera I think that this line should be "when you **are** adding **a** new migration to **an** existing app..."

File: source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    drop_table :posts
+    Post.drop_translation_table!
+  end
+end
+```
+**Note** _that the ActiveRecord model Post must already exist and have listed attributes for translations_.
+
++ and run `rake db:migrate` .
+
+Problem come when you change your mine and deside to leave title to be untranslatable.
```
Comment (JovicaSusa) [inline source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown]: @ikera I think that here you should have "Problem come**s** when you change your min**d**.."

Comment (marijanam) [inline source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown]: and de**c**ide

File: source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+Here it is! If you decide to delete your database and create it again you need to use:
+
++ `rake db:drop`
++ `rake db:create`
++ `rake db:schema:load`
+
+Because, if you try to use `rake db:migrate` instead of `rake db:schema:load` you will get **BIG ERROR!**, because for your first migration "create_posts" it is necessary that you have defined translatable attributes :title and :text in Post model, but you removed :title from Post model translations.
+
+So just fallow above [advice](#advice), and good luck.
```
Comment (JovicaSusa) [inline source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown]: @ikera "just f**o**llow advice above.." maybe it's better to switch above and advice

File: source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown [hunk 6]
```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "rake db:schema:load vs rake db:migrate"
+date: 2016-03-23 08:09:25 +0100
```
Comment (basti) [inline source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown]: @Ikera before merging to master and publishing make sure you update date/time to today and rename the file too.

File: source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+  - globalize
+published: true
+---
+
+Sooner or later every new Ruby developer needs to understand  differences between this two common rake tasks. Basically, these simple definition tells us everything we need to know:
+
++ `rake db:migrate` runs migrations that have not run yet
++ `rake db:schema:load` loads the schema.db file into database.
+
+but the right question is when to use one or the other.
```
Comment (basti) [inline source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown]: @Ikera maybe `but the real question` would sound better.

File: source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+Here it is! If you decide to delete your database and create it again you need to use:
+
++ `rake db:drop`
++ `rake db:create`
++ `rake db:schema:load`
+
+Because, if you try to use `rake db:migrate` instead of `rake db:schema:load` you will get **BIG ERROR!**, because for your first migration "create_posts" it is necessary that you have defined translatable attributes :title and :text in Post model, but you removed :title from Post model translations.
+
+So just fallow [advice](#advice) above, and good luck.
```
Comment (basti) [inline source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown]: @Ikera spelling: `fallow -> follow`

**Comment by basti** [comment]
:+1:

**Comment by draganf** [comment]
:+1:

**Pull request merged**